content/release-10-15-0/announcements-10-15-0.md

```diff
@@ -194,3 +194,15 @@ This is available from the following web page: https://documentation.softwareag.
 Further to the CEL (Esper) deprecation notice in [release 10.5](/release-10-5-0/migration-10-5-0/) and the subsequent announcement of end of support in [release 10.7](/release-10-7-0/announcements-10-7-0/) it has been possible to continue using Esper in an unsupported mode. With effect from release 10.15 the CEL (Esper) functionality has been removed completely and cannot be used anymore.
 
 If you need assistance to migrate your streaming analytics logic from Esper to Apama, please contact [Software AG Global Support](/about/contacting-support/).
+
+### Cumulocity IoT DataHub
+
+#### Planned
+
+##### Breaking change in the offloading mechanism - support for mixed types will be deprecated
+
+In a future release of Cumulocity IoT DataHub a newer version of Dremio will be adopted that removes support for mixed data types in a single column. The offloading process will be adapted so that mixed types will be automatically resolved during offloading. Cumulocity IoT DataHub will not be able to read data lake contents containing mixed data types, so corresponding data needs to be converted.
```

Comment:
Do we need to add a link where conversion of the data is explained? Or is this obvious?

Reply:
Good idea, but so far this is not completely clear how that conversion will look like

Reply:
So perhaps you should add a similar sentence as below?

Reply:
We deliberately kept it vague until we have fleshed out more details of that conversion. So I'd tend to keep it like that.